src/code/PublishPSResource.cs

```diff
@@ -575,7 +575,8 @@ protected override void EndProcessing()
                         }
 
                         // defaults to false
-                        string requireLicenseAcceptance = psData.ContainsKey("requirelicenseacceptance") ? psData["requirelicenseacceptance"].ToString() : "false";
+                        // boolean value needs to be a lowercase string for it to be correctly parsed

```

Comment:
Just a nitpick in case this sounds clearer
```suggestion
                        // Value for requireAcceptLicense key needs to be a lowercase string representation of the boolean for it to be correctly parsed from psData file.

```

Reply:
Or what you have in the PR description sounds a bit clearer as a requirement too :)

"RequireLicenseAcceptance value is a string that needs to be lowercase in order for it to later be converted into a boolean value correctly"